modules/installation-infrastructure-user-infra.adoc

```diff
@@ -74,6 +74,13 @@ endif::ibm-z-kvm[]
 . Ensure that your network infrastructure provides the required network connectivity between the cluster components. See the _Networking requirements for user-provisioned infrastructure_ section for details about the requirements.
 
 . Configure your firewall to enable the ports required for the {product-title} cluster components to communicate. See _Networking requirements for user-provisioned infrastructure_ section for details about the ports that are required.
++
+[IMPORTANT]
+====
+By default, port `1936` is accessible for a {product-title} cluster, because each control plane node needs access to this port. 
```

Comment:
```suggestion
By default, the port `1936` is accessible for an {product-title} cluster, because each control plane node needs access to this port. 
```